plutus-core/src/Language/PlutusCore/Parser.y

```diff
@@ -174,35 +174,35 @@ getStaticBuiltinName = \case
 --- Parsing built-in types and constants ---
 
 -- | Tags of types in the default universe.
-tagOfTyName :: T.Text -> Maybe Int
-tagOfTyName = \case
-    "bool"       -> Just $ tagOf DefaultUniBool
-    "bytestring" -> Just $ tagOf DefaultUniByteString
-    "char"       -> Just $ tagOf DefaultUniChar
-    "integer"    -> Just $ tagOf DefaultUniInteger
-    "string"     -> Just $ tagOf DefaultUniString
-    "unit"       -> Just $ tagOf DefaultUniUnit
+encodeTyName :: T.Text -> Maybe [Int]
+encodeTyName = \case
+    "bool"       -> Just $ encodeUni DefaultUniBool
+    "bytestring" -> Just $ encodeUni DefaultUniByteString
+    "char"       -> Just $ encodeUni DefaultUniChar
+    "integer"    -> Just $ encodeUni DefaultUniInteger
+    "string"     -> Just $ encodeUni DefaultUniString
+    "unit"       -> Just $ encodeUni DefaultUniUnit
```

Comment:
Ultimately we don't want this to be coded into the Parser, but somehow done by something passed to the parser as a parameter, don't we?  Will that be in a separate PR?

Reply:
Yes, that's for a different PR.